### DISCUSSION TITLE
Bisakah Kustom Feed XML dan Sitemap pada Tampilan Page?

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#258] [Q&A]
Kang, bagaimana cara custom url Feed XML dan Sitemap pada tampilan Page ? bisakah url Feed dan sitemap parentnya aja yang muncul di tampilan Pagenya kang?

**Comment by taufik-nurrohman** [comment]
Daripada kustom tampilan di mode laman tunggal mending cari cara buat nampilin elemen `<link href>` yang mengarah ke RSS laman indeks induknya ketika kita ada di mode laman tunggal.

**Reply by yuliakbard** [reply]
yap benar kang, ini lagi cari cara buat ngakalin url nya biar muncul parent->url nya aja

**Reply by yuliakbard** [reply]
pake Hook::set ya kang di file layout? trus dikondisionalkan gitu ya kang?

**Reply by yuliakbard** [reply]
`$url->path` nya ganti jadi `URL::short($parent->url)` ini yang terpikirkan sama ane kang

**Reply by yuliakbard** [reply]
```
if ('sitemap.xml atau feed.xml' === basename($url->path ?? "")) {
    Hook::set('content', function ($content) {
if($site->is('page')){
//...
}else {
//...
}
//...
    }, 20);
}
```

pake ini ya kang?

**Reply by taufik-nurrohman** [reply]
Iya. Supaya nggak perlu ngedit berkas ekstensi coba sekali-kali pakai fitur `Hook::let()` untuk melepas kait yang sudah dipasang.
